Tumbleweed

qiangzhao opened the pull request:
Remove plymouth watermark link from branding package to avoid directories not owned error after resort plymouth sub-packages dependence. And the watermark determination will move to plymouth source.

hellcp commented:
Would you fix it so this pr is just one commit instead of having those 3 additional commits from years ago? Thanks!

qiangzhao committed:
Remove plymouth watermark link from branding package to
avoid directories not owned error after resort plymouth
sub-packages dependence. And the watermark determination
will move to plymouth source.

qiangzhao commented:
> Would you fix it so this pr is just one commit instead of having those 3 additional commits from years ago? Thanks!
@hellcp  Have fixed. thank you!

qiangzhao commented:
Thank you Sasi!
